samples/highcharts/demo/bellcurve/demo.js

```diff
@@ -1,62 +1,257 @@
-const createChart = data => Highcharts.chart('container', {
+function createChart(data, getJitterData) {
+    Highcharts.chart('container', {
 
-    title: {
-        text: 'Bell curve'
-    },
-
-    xAxis: [{
-        title: {
-            text: 'Data'
-        },
-        alignTicks: false
-    }, {
         title: {
             text: 'Bell curve'
         },
-        alignTicks: false,
-        opposite: true
-    }],
 
-    yAxis: [{
-        title: { text: 'Data' }
-    }, {
-        title: { text: 'Bell curve' },
-        opposite: true
-    }],
+        xAxis: [{
+            title: {
+                text: 'Data value'
+            },
+            alignTicks: false
+        }],
 
-    series: [{
-        name: 'Bell curve',
-        type: 'bellcurve',
-        xAxis: 1,
-        yAxis: 1,
-        baseSeries: 1,
-        zIndex: -1
-    }, {
-        name: 'Data',
-        type: 'scatter',
-        data: data,
-        accessibility: {
-            exposeAsGroupOnly: true
-        },
-        marker: {
-            radius: 1.5
-        }
-    }]
-});
+        yAxis: [{
+            title: { text: 'Probability density' }
+        }, {
+            visible: false,
+            min: -1,
+            max: 1
+        }],
+
+        series: [{
+            name: 'Bell curve',
+            type: 'bellcurve',
+            data: data,
+            zIndex: -1,
+            enableMouseTracking: false
+        }, {
+            type: 'scatter',
+            data: getJitterData(data),
```

Comment:
Instead of passing over the `getJitterData` function I think we should just do the `map` inline. With a short comment explaining why we set y = 0 on all points.